sdk/dotnet/Iam/Role.cs

```diff
@@ -204,6 +204,8 @@ namespace Pulumi.Aws.Iam
     /// 
     /// ### Example of Exclusive Managed Policies
     /// 
+    /// &gt; The `managed_policy_arns` argument is deprecated. Use the `aws.iam.RolePolicyAttachmentsExclusive` resource instead.
```

Comment:
Hmm.

Reply:
I feel like this is actually a good thing. The inline `managedPolicyArns` are a major foot gun.

I feel like by not surfacing that deprecation we might create more pain when the next major comes around and they possibly remove it (not sure that's upstream's plan though)

Reply:
Discussing internally.